front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -59,83 +51,19 @@ const CustomFieldTitle = styled.div`
   font-weight: 500;
 `;
 
-const ChevronIcon = styled(Icon)`
-  width: 12px;
-  height: 12px;
-  fill: ${colors.label};
-  margin-left: 20px;
-  transition: fill 80ms ease-out, transform 200ms ease-out;
+const StyledAccordion = styled(Accordion)`
+  border-bottom: none;
 `;
 
-const CollapsedContent = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
+const HeadingContainer = styled.div`
```

Comment:
Instead of this - can we use the appropriate `Heading` from the component library?

Reply:
Do we have a Heading component? I can't seem to find one. I think we can use `Box` here as-well

Reply:
Sorry - my bad, I meant `Title` component. It's important to not use Box-es for everything as we are trying to keep the html semantic :)

Reply:
Sorry - my bad, I meant `Title` component. It's important to not use Box-es for everything as we are trying to keep the html semantic :)